src/assets/scripts/client/aircraft/AircraftCommander.js

```diff
@@ -290,15 +290,15 @@ export default class AircraftCommander {
         const turnDirection = data[0];
         const legLength = data[1];
         const holdFix = data[2];
-        let holdFixLocation = aircraft.position;
+        let holdPosition = aircraft.position;
         let inboundHdg = aircraft.heading;
 
         if (holdFix) {
-            holdFixLocation = this._navigationLibrary.getFixPositionCoordinates(holdFix);
-            inboundHdg = bearingToPoint(aircraft.position, holdFixLocation);
+            holdPosition = this._navigationLibrary.findFixByName(holdFix).position;
```

Comment:
I was actually confused by the two instances I came across where you were encapsulating the position in an object like this, as that was what was causing a few of the failing tests. `{position: Position}` is not what should be returned anyway. It should be a straight `PositionModel`.